### PR TITLE
Python 3: To translate raise

### PR DESCRIPTION
**Pull request by hsmj1412**
Converts raise E, V to raise E(V),
and raise E, V, T to six.reraise(E, V, T)

Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

File: avocado_vt/test.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def verify_background_errors(self):
         except Queue.Empty:
             pass
         else:
-            raise exc[1], None, exc[2]
+            raise exc[1].with_traceback(exc[2])
```
Comment (chunfuwen) [inline avocado_vt/test.py]: need introduce six module to keep compatable:
-raise E, V, T
+mport six
+six.reraise(E, V, T)

File: avocado_vt/test.py [hunk 2]
```diff
@@ -60,6 +60,9 @@
 from autotest.client.shared import error
```
Comment (chunfuwen) [inline avocado_vt/test.py]: You may take this opportunity to remove dependency for  from autotest.client.shared import error

File: avocado_vt/test.py [hunk 3]
```diff
@@ -263,22 +239,15 @@ def runTest(self):
...
         """
         if self.__status != "PASS":
-            if isinstance(self.__status, error.TestNAError):
-                self.cancel(str(self.__status))
-            elif isinstance(self.__status, error.TestWarn):
-                self.log.warn(str(self.__status))
-            elif isinstance(self.__status, error.TestFail):
-                self.fail(str(self.__status))
-            else:
-                raise self.__status  # pylint: disable=E0702
```
Comment (kylazhang) [inline avocado_vt/test.py]: why remove the judgement here?

Comment (chunfuwen) [reply avocado_vt/test.py]: runtest()actually not called in this py file,but _runtest() was used instead, not quite sure whether it is useful or not. So better change them to exceptions.TestError,exceptions.TestFail, exceptions.TestWarn respectively

Comment (hsmj1412) [reply avocado_vt/test.py]: As autotest has been obsoleted, so this section is deleted. But some tp-libvirt tests still need autotest, freeze this pr until tp-libvirt update or freeze the avocado-vt tag

**Pull request merged**